### PR TITLE
Debugging publishing

### PR DESCRIPTION
**Pull request by rkeur7**
Link to ticket: https://shippo.atlassian.net/browse/CET-344

### CODE REVIEW

File: .github/workflows/sdk_publish.yaml [hunk 1]
```diff
@@ -14,13 +14,44 @@ on:
...
-      ossrh_password: ${{ secrets.OSSRH_PASSWORD }}
-      ossrh_username: ${{ secrets.OSSRH_USERNAME }}
-      speakeasy_api_key: ${{ secrets.SPEAKEASY_API_KEY }}
+    runs-on: ubuntu-latest
+    env:
+      JRELEASER_GPG_PASSPHRASE: ${{ secrets.SONATYPE_PASSPHRASE }}
+      JRELEASER_GPG_SECRET_KEY: ${{ secrets.SONATYPE_SECRET_KEY }}
+      JRELEASER_DEPLOY_MAVEN_MAVENCENTRAL_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+      JRELEASER_DEPLOY_MAVEN_MAVENCENTRAL_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
+      JRELEASER_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (rkeur7) [inline .github/workflows/sdk_publish.yaml:23]: Speakeasy's tooling uses OSSRH, Maven Central's new offering is not included in Speakeasy's tooling. Maven Central's recommended plugins are here: https://central.sonatype.org/publish/publish-portal-gradle/

**Comment by rkeur7** [comment]
[Here are publishing results using dry-run](https://github.com/goshippo/shippo-java-sdk/actions/runs/12057764593). This needs to be merged to test it fully.

File: README.md [hunk 2]
```diff
@@ -79,8 +85,8 @@ gradlew.bat publishToMavenLocal -Pskip.signing
 ```java
 package hello.world;
 
-import com.goshippo.shippoSDK.Shippo;
-import com.goshippo.shippoSDK.models.operations.ListAddressesResponse;
+import com.goshippo.shippo_sdk.Shippo;
```
Comment (michellemiller-shippo) [inline README.md:88]: This looks like a breaking / painful change to customers. I'm just thinking of the javax > jakarta changes, even when it was simple it was so annoyingly not backwards-compatible =/ our Address API has gotten customer complaints for similar casing changes, even on breaking versions

File: docs/sdks/batches/README.md [hunk 3]
```diff
@@ -0,0 +1,660 @@
...
+* [get](#get) - Retrieve a batch
+* [addShipments](#addshipments) - Add shipments to a batch
+* [purchase](#purchase) - Purchase a batch
+* [removeShipments](#removeshipments) - Remove shipments from a batch
+
+## create
+
+Creates a new batch object for purchasing shipping labels for many shipments at once. Batches are created asynchronously. This means that the API response won't include your batch shipments yet. You need to retrieve the batch later to verify that all batch shipments are valid.
+
+### Example Usage
```
Comment (michellemiller-shippo) [inline docs/sdks/batches/README.md:29]: I feel like these examples would be more useful in separate Java files if not a separate sample Shippo repo with a URL

File: README.md (hunk 2 above)
Comment (rkeur7) [reply README.md:88]: This isn't published yet so nobody is using it yet 😄

**Pull request merged**